package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "causal-tree-ts",
+  "version": "1.0.0",
+  "description": "Typescript implementation of causal tree (CRDT)",
+  "main": "index.js",
+  "scripts": {
+    "dev": "npx nodemon --watch src/ --exec npx ts-node -r dotenv/config src/app.ts -e ts",
+    "test": "npx jest --watch",
+    "test:cov": "npx jest --coverage"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/crdteam/causal-tree-ts.git"
+  },
+  "keywords": [
+    "crdt",
+    "causal",
+    "tree",
+    "node"
+  ],
+  "author": "@samuelv8",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/crdteam/causal-tree-ts/issues"
+  },
+  "homepage": "https://github.com/crdteam/causal-tree-ts#readme",
```

Comment:
It'd be useful to add a barebones README containing main commands, like how to install, lint, test, etc.

Reply:
Also, please add any version limits that are not captured here, like npm and node themselves.

I'm a TS newb and I've just installed an environment with the following, and my package.json copy was modified:

```sh
nvm install 18
nvm use 18
npm install
npx jest
```